src/js/components/AlertDialogErrorProvider/AlertDialogErrorProvider.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { 
+    AlertDialog, 
+    AlertDialogBody, 
+    AlertDialogContent, 
+    AlertDialogFooter, 
+    AlertDialogHeader, 
+    AlertDialogOverlay,
+    Button,
+} from '@chakra-ui/react';
+import { IProviderProps } from '../../types/Types';
+import useGlobalSettings from '../../GlobalSettings/useGlobalSettings';
+import { IAlertDialogErrorProvider } from '../../types/AlertDialogProvider/AlertDialogProvider';
+
+
+
+const DEFAULT_ALERT_DIALOG_CONTEXT_VALUE: IAlertDialogErrorProvider = {
+    alertDialogError: (header: string, alertMessage: string, buttonMessage: string) => {}
+}
+
+export const AlertDialogContext = React.createContext<IAlertDialogErrorProvider>(DEFAULT_ALERT_DIALOG_CONTEXT_VALUE);
+
+
+
+function AlertDialogError({children}: IProviderProps): JSX.Element {
+
+    const [isOpen, setIsOpen] = React.useState<boolean>(false);
+    const closeButtonRef = React.useRef<any>();
```

Comment:
Don't use `any` type

Reply:
@NikitaGuyAvrahamCom also add eslint here with rule that will avoid any type
@yehorGuyAvrahamCom can show u where we have it ready. talk to him on slack